lib/ziti-tunnel-cbs/ziti_tunnel_ctrl.c

```diff
@@ -225,12 +225,12 @@ static int process_cmd(const tunnel_comand *cmd, command_cb cb, void *ctx) {
             }
             if (inst) {
                 disconnect_identity(inst->ztx, CMD_CTX.tunnel_ctx);
-                model_map_remove(&instances, disable_id.path);
+                model_map_remove(&instances, disable_id.identifier);
                 result.success = true;
             } else {
                 result.success = false;
-                result.error = calloc(1, strlen(disable_id.path) + 35);
-                sprintf(result.error, "ziti instance for id %s is not found", disable_id.path);
+                result.error = calloc(1, strlen(disable_id.identifier) + 35);
```

Comment:
will this ever get freed? I think we had a similar issue in a previous PR.

Reply:
removing this calloc